docs/usage_general.rst.inc

```diff
@@ -261,6 +261,8 @@ Please note:
 
 .. _INI: https://docs.python.org/3/library/logging.config.html#configuration-file-format
 
+.. _tempfile: https://docs.python.org/2/library/tempfile.html
+
```

Comment:
please use py 3 docs and if it is specifically about the platform docs, maybe say so and link there?

https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir

Reply:
Uh yes, sorry sloppiness on my side. I didn't link to the anchor on purpose, because I saw so many paragraphs speaking about TMPDIR, but actually the link you provided is the right paragraph (and the others are not).